Framework/API/inc/MantidAPI/WorkspaceGroup.h

```diff
@@ -124,6 +124,9 @@ class MANTID_API_DLL WorkspaceGroup : public Workspace {
   WorkspaceGroup *doClone() const override {
     throw std::runtime_error("Cloning of WorkspaceGroup is not implemented.");
   }
+  WorkspaceGroup *doCloneEmpty() const override {
+    throw std::runtime_error("Cloning of WorkspaceGroup is not implemented.");
```

Comment:
`WorkspaceGroup` should just be a container around a `std::vector<Workspace_sptr>`